libs/auth/react/src/lib/utils/toStringScope.spec.ts

```diff
@@ -0,0 +1,12 @@
+import { toStringScope } from './toStringScope'
+
+describe('toStringScope', () => {
+  it('should return a string for an array of scope', () => {
+    expect(toStringScope(['scope1', 'scope2'])).toBe('scope1 scope2')
+  })
+
+  it('should return an empty string when no scope is defined', () => {
+    expect(toStringScope()).toBe('')
+    expect(toStringScope([])).toBe('')
```

Comment:
might want an extra case for the empty scope - not sure how ids or other clients might handle a scope like `['something', '', 'something']`? throw error? maybe not the correct place for a check like that...?

Reply:
It would mean in the first place is has been defined as an empty string. That should not happen, right?

Reply:
probably not, just a minor think about it, if it's not a concern just disregard my comment